packages/client/hmi-client/src/utils/color-schemes.ts

```diff
@@ -0,0 +1,16 @@
+export const strataTypeColors = [
+	'#fde725',
+	'#cde11d',
+	'#98d83e',
+	'#67cc5c',
+	'#40bd72',
+	'#25ac82',
+	'#1f998a',
+	'#24878e',
+	'#2b748e',
+	'#34618d',
+	'#3d4d8a',
+	'#453581',
+	'#481c6e',
+	'#440154'
+];
```

Comment:
Did you get the palette some where? if so can you leave a link? 
If you create this yourself can you give a brief description/comment of the characteristics of this palette?

Reply:
This is just temporary, I used the same ones for the simulation graph. I'll add a comment explaining as such.

Reply:
Okay, If you mean you used VIRDIS colour palette that is probably not the right solution. VIRDIS is meant for a continuous range of values. Here, with the graph colouring, what we would rather have is a categorical colouring palette. E.g. Susceptible is twice as high as Recovered makes no sense.

Colorbrewer here https://colorbrewer2.org/#type=sequential&scheme=BuGn&n=3a provide some nice categorical (qualitative) palette you can select.